### PR TITLE
[12.0][FIX] base_jsonify: In odoo 12, Date and Datetime fields values are d…

### PR DESCRIPTION
**Pull request by lmignon**
…atetime objects

Convert fields.Date and fields.Datetime values to string when serializing to json

### CODE REVIEW

**Review by simahawk — APPROVED** [review]

**Review by acsonefho — APPROVED** [review]
Code LGTM

**Review by rousseldenis — APPROVED** [review]

File: base_jsonify/models/models.py [hunk 1]
```diff
@@ -71,6 +71,10 @@ def jsonify(self, parser):
                     value = rec[field_name]
                     if value is False and field_type != 'boolean':
                         value = None
+                    elif field_type == "date":
+                        value = fields.Date.to_string(value)
+                    elif field_type == "datetime":
+                        value = fields.Datetime.to_string(value)
```
Comment (hparfr) [inline base_jsonify/models/models.py:77]: I think it's better to be compliant with ISO 8601 here.
Date is ok,  but date_time defined here https://github.com/odoo/odoo/blob/12.0/odoo/tools/misc.py#L568
is with a space as separator between dates and time. Plus it's lacking timezone.

https://www.w3.org/TR/NOTE-datetime

Comment (sbidoul) [reply base_jsonify/models/models.py:77]: @lmignon have you seen this comment? Sounds important to me too.

Comment (lmignon) [reply base_jsonify/models/models.py:77]: @sbidoul Yes I see it, I'll provide a new PR to fix this.

**Pull request merged**